lib/flight_scheduler/event_processor.rb

```diff
@@ -70,7 +70,11 @@ def allocate_resources_and_run_jobs
       end.join("\n")
     }
     Async.logger.debug("Allocated nodes") {
-      allocated_nodes = FlightScheduler.app.allocations.each.map { |a| a.nodes }.flatten.sort.uniq
+      allocated_nodes = FlightScheduler.app.allocations.each
+        .map { |a| a.nodes }
```

Comment:
Generally I would align this with the first `.` however its only a stylistic/ re-ability change.

Also whilst I am being a :panda_face:, this is using an `each.map`. I know this pre-existed the change, but just saw it

Reply:
The allocation registry doesn't include `Enumerable`.  If it did, methods such as `drop` and `drop_while` would be available.  Perhaps that would be fine, or perhaps not.  I've deliberately avoid thinking about that.  Instead I opted to require users to use `each.<enumerable_method>`.  This is syntactic vinegar, which should cause the programmer to stop and think about whether they ought to be doing that or not.  In this particular case, I decided that `each.map` was OK for a debug message.